distribution/changelog.txt

```diff
@@ -1,5 +1,6 @@
 0.0.5 (in development)
 ------------------------------------------------------------------------
+- Feature: Implementation of the user-defined currency
 - Feature: Add OpenGL drawing engine.
```

Comment:
OpenGL is bigger and something that should be on the top, the thing is that I put it above it to sort it as 'latest thing that got merged in order'